bpf/init.sh

```diff
@@ -93,8 +93,8 @@ ip addr add $HOST_IP/128 dev $HOST_DEV1
 
 ip route del $ADDR/128 dev $HOST_DEV1 2> /dev/null || true
 ip route add $ADDR/128 dev $HOST_DEV1
```

Comment:
Just review this part again, we are using `:ffff` on the Linux side and `:0000` in the virtual L3 space where the endpoints are connected.

```HOST_IP=$(echo $ADDR | sed 's/:0$/:ffff/')```

We are also deleting the automatically created route for the `/128` in the local table and replace it with a scope global one. This part:
```
ip route del $ADDR/128 dev $HOST_DEV1 2> /dev/null || true
ip route add $ADDR/128 dev $HOST_DEV1
```

 I remember doing this because Linux would refuse to route into the veth if the address is a known local address. However, it seems to work fine on the IPv4 side so this may in fact be something we can get rid of after some testing.